oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/spi/binary/FulltextBinaryTextExtractor.java

```diff
@@ -125,7 +131,13 @@ public void done(boolean reindex){
   private String parseStringValue(Blob v, Metadata metadata, String path, String propertyName) {
     String text = extractedTextCache.get(path, propertyName, v, reindex);
     if (text == null){
+      textExtractionTimerMetricStats = (extractedTextCache.getStatisticsProvider() != null) ? extractedTextCache.getStatisticsProvider().
+              getTimer(TEXT_EXTRACTION_TIMER_METRIC_NAME, StatsOptions.METRICS_ONLY) : null;
       text = parseStringValue0(v, metadata, path);
+      if (textExtractionTimerMetricStats != null) {
+        TimerStats.Context context = textExtractionTimerMetricStats.time();
+        context.stop();
+      }
```

Comment:
Ehm, I think this is a bug... doesn't "textExtractionTimerMetricStats.time();" start the timer, and "context.stop()" stop it? So there should be some something you measure the time for between the two lines...

Reply:
Maybe you only care about the count (number of times it's executed). But in that case, I wonder if there is a simpler way to do this.

Reply:
added parseStringValue0 in between start and stop